extensions/spark/kyuubi-spark-connector-hive/src/test/scala/org/apache/kyuubi/spark/connector/hive/HiveQuerySuite.scala

```diff
@@ -186,4 +189,19 @@ class HiveQuerySuite extends KyuubiHiveTest {
       }
     }
   }
+
+  test("read partitioned avro table") {
```

Comment:
let's also add test for un-partitioned serde table

Reply:
Done, and add other formats.